app/policies/mhv_messaging_policy.rb

```diff
@@ -27,6 +27,17 @@ def validate_client(client)
       true
     end
   rescue
+    log_denial_details(client)
     false
   end
+
+  def log_denial_details(client)
+    Rails.logger.info('SM ACCESS DENIED IN MOBILE POLICY',
```

Comment:
Would it make sense to also update the log message for Rx, too? https://github.com/department-of-veterans-affairs/vets-api/blob/9249ac1e32b01e0e8191054220ac83dbbc03246b/app/policies/mhv_prescriptions_policy.rb#L13

Or are we talking about different things?

Reply:
SM and RX policies are different. We eventually want to update RX in a similar way to SM but there's no equivalent API that tells us if someone does or doesn't have RX access atm.